packages/features/ee/workflows/lib/reminders/smsReminderManager.ts

```diff
@@ -70,7 +71,10 @@ export const scheduleSMSReminder = async (
   async function getIsNumberVerified() {
     if (action === WorkflowActions.SMS_ATTENDEE) return true;
     const verifiedNumber = await prisma.verifiedNumber.findFirst({
-      where: { userId, phoneNumber: reminderPhone || "" },
+      where: {
```

Comment:
phone number either needs to be verified by the user or the team the workflow belongs to